app/controllers/devise/devise_authy_controller.rb

```diff
@@ -14,6 +14,10 @@ class Devise::DeviseAuthyController < DeviseController
 
   def GET_verify_authy
     @authy_id = @resource.authy_id
+    if ENV.fetch('ENABLE_AUTHY_ONETOUCH', false)
```

Comment:
I think it's better to use devise configuration for consistency. 
[Here's an example](https://github.com/authy/authy-devise/blob/master/lib/devise-authy/controllers/helpers.rb#L16)
Here you can see that we use them with the convention `authy_XXX`, it could be `authy_onetouch_enabled`

Reply:
Done. I can't run tests locally (I get a bunch of `uninitialized constant AbstractController::Rendering::Activity` when controllers render in tests for some reason), so let me know if anything needs fixing.